CHANGELOG.md

```diff
@@ -244,6 +244,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
     * Removed: keeper `GetConstantFee`, `SetConstantFee` 
 * (x/mint) [#16329](https://github.com/cosmos/cosmos-sdk/pull/16329) Use collections for state management:
   * Removed: keeper `GetParams`, `SetParams`, `GetMinter`, `SetMinter`.
+* (x/evidence) [#16336](https://github.com/cosmos/cosmos-sdk/pull/16336) Use collections for state management:
```

Comment:
evidence has its own changelog we should put this there

Reply:
thanks, resolved!